azure/terraform.sh

```diff
@@ -75,6 +75,16 @@ function other_actions() {
   fi
 }
 
+## Apply state don't have -var-file or vars because are inherith from output state
+function apply_state_action() {
```

Comment:
1) tbh, I didn't get what this function does and what an "output state" is.
2) Furthermore, there is no need to create another function. You should use the already present "other_actions" function just adding another case (via "if" or "case" statement)
3) Lastly, you should increment the "VERSION" variable at the beginning of this script, otherwise it's not possible to know what script is the last version of it

Reply:
ok i will change the others function